src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+/* code here */
+@import "../utils/variables";
+
+.product {
+  display: flex;
+  box-sizing: border-box;
```

Comment:
box sizing should be the same for all elements, move it to `*` selector in normalize